package.json

```diff
@@ -20,6 +20,7 @@
         "build:external": "npx lerna run build --stream --ignore '{@wso2is/console,@wso2is/myaccount}' && npx lerna run build:external --stream --scope '{@wso2is/console,@wso2is/myaccount}'",
         "clean": "npx lerna run clean --stream",
         "clean-all": "npm run remove-package-lock && npm run remove-maven-folders && npm run remove-node-modules",
+        "postinstall": "node ./scripts/post-install.js",
```

Comment:
Let's put this file to the extensions folder

Reply:
Isn't this better to be global since used in both console and myaccount?

Reply:
Removed due to the issue as discussed